modules/project-docs/pages/compatibility.adoc

```diff
@@ -60,9 +60,8 @@ The PHP SDK is built on top of libcouchbase (the C SDK), which is tested and sup
 === GNU/Linux
 
 * Amazon Linux 2.
-* Red Hat Enterprise Linux 7 & 8;
-+
-& CentOS 7 & 8.
+* Red Hat Enterprise Linux 7 (Server only -- see Capella note below) & RHEL 8.
```

Comment:
"Server only" feels a bit unclear, and the Capella note describes the issue anyway, so could simply be omitted.

Split onto multiple lines?
(That way, the unproblematic CentOS 8 and RHEL 8 are more noticeable, rather than tucked at the end of of a lot of faffing)

If the NOTE can't be moved up to sit contiguous with this section, as suggested, then is it worth xref:ing it?